tiny_dnn/core/framework/tensor.h

```diff
@@ -41,6 +41,14 @@ class Tensor {
   Tensor() {}
 
   /**
+   * Constructor that accepts a storage object and create a Tensor housing
+   * that storage. Commonly used by [] operator.
+   * @param storage object, can be xtarray.
+   * @return
+   */
+  explicit Tensor(Storage s) : storage_(s) {}
```

Comment:
@edgarriba C++ is a strongly typed language. In this case, the argument type name `Storage` represents its characteristics so renaming the argument to `storage` does not give much benefit. Moreover, its variable scope is quite short.

Having more context isn't always good. Less is More. Controlling pace and accent is important.